src/tailwind.js

```diff
@@ -563,6 +564,7 @@ module.exports = {
     '24': '6rem',
     '28': '7rem',
     '30': '7.5rem',
+    '40': '9rem',
```

Comment:
Shouldn't this be 10rem? If you're trying to add 9rem, you want 36 (4 * 9) instead of 40